x/gamm/pool-models/balancer/amm_joinpool_test.go

```diff
@@ -798,13 +831,14 @@ func TestCalcSingleAssetJoin(t *testing.T) {
 
 func TestCalcJoinSingleAssetTokensIn(t *testing.T) {
 	testCases := []struct {
-		name           string
-		swapFee        sdk.Dec
-		poolAssets     []balancer.PoolAsset
-		tokensIn       sdk.Coins
-		expectShares   sdk.Int
-		expectLiqudity sdk.Coins
-		expErr         error
+		name                 string
```

Comment:
can we make this be `calcJoinSharesTestCase`? Seems that most of the fields are the same